english/talk_to_neighbour/woman/say-hi/rare.md

```diff
@@ -1,5 +1,6 @@
-The alien seems surprised that you said hello and replies: 
+The alien seems surprised that you said hello and replies:
 "Thank you! Very few humans have been polite enough to acknowledge my existance at all."
 "Because you were nice, I will increase your intelligence by 10%"
 Suddenly, you feel 10% sadder.
 
+However, you realize you've seen this alien before. You're pretty sure this was one of the aliens from "Independence Day," starring Will Smith.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.